renovate.json5

```diff
@@ -30,7 +30,9 @@
   },
 
   postUpgradeTasks: {
-    commands: ['make bs _bootstrap-doc-tools build'],
-    fileFilters: ['dist/**/*', 'documentation/**/*'],
+    // NOTE(dweitzman)): _bootstrap-doc-tools doesn't work in the renovate container
+    // at the moment so we skip building docs and just compile code.
```

Comment:
this doesn't affect building docs in main branch or PRs right?

Reply:
No, the other builds on the repo's circleci won't be affected. This configuration is only used by Renovate while making PRs to upgrade dependencies, which is a separate circleci. A failure here also doesn't stop Renovate from making a PR, it just makes a PR with a comment saying that the artifacts may not be correct